src/common/productReleaseEntityAttributes.ts

```diff
@@ -20,6 +20,7 @@ export const productReleaseEntityAttributes = {
     filter: {
       operator: '=like=',
     },
+    tooltip: "Release part of the Product Version, e.g. '0.GA' for Product ersion '1.0'.",
```

Comment:
typo
`tooltip: "Release part of the Product Version, e.g. '0.GA' for Product version '1.0'.",`